subsys/net/lib/hostap_crypto/Kconfig

```diff
@@ -0,0 +1,140 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+choice WIFI_NM_WPA_SUPPLICANT_CRYPTO_BACKEND
+	default WIFI_NM_WPA_SUPPLICANT_CRYPTO_EXT
+endchoice
+
+choice HOSTAP_CRYPTO_BACKEND
```

Comment:
My initial implementation used the same prefixes as WIFI_NM_WPA_SUPPLICANT_*, but to avoid confusion renamed this, I also need to update docs, but raising this early for review comments.